build/template-run-unit-tests.yaml

```diff
@@ -3,18 +3,57 @@
 
 steps:
 
+- task: VSTest@2
+  displayName: 'Run integration tests (.NET 8)'
+  inputs:
+    testSelector: 'testAssemblies'
+    testAssemblyVer2: 'Tests\**\WebAppUiTests\**\net8.0\WebAppUiTests.dll'
+    searchFolder: '$(System.DefaultWorkingDirectory)'
+    rerunFailedTests: true
+    rerunMaxAttempts: '3'
+    runInParallel: false
+    codeCoverageEnabled: true
+    failOnMinTestsNotRun: false
+    minimumExpectedTests: '1'
+  continueOnError: true
```

Comment:
Hmm, but you have to figure out a way to break the build if the integration tests fail.

Reply:
This was for testing. removed